lib/bolt/bolt_option_parser.rb

```diff
@@ -263,9 +267,19 @@ def self.examples(cmd, desc)
         createkeys               Create new encryption keys
         encrypt                  Encrypt a value
         decrypt                  Decrypt a value
+
       Available options are:
     SECRET_HELP
 
+    INVENTORY_HELP = <<~INVENTORY_HELP
+      Usage: bolt inventory <action>
+
+      Available actions are:
+        show                     Show list of targets in an inventory group
```

Comment:
`Show the list of targets an action would run on` or something? Since it's supports `query` and `rerun` I think that makes the most sense.